docs-2.0/k8s-operator/4.cluster-administration/4.7.security/4.7.1.enable-mtls.md

```diff
@@ -493,34 +502,92 @@ Details about these fields are as follows:
 
 ### Configure `sslCerts`
 
-When you enable mTLS between services, you still needs to set `spec.sslCerts`, because NebulaGraph Operator communicates with the Meta service and Storage service. 
+As NebulaGraph Operator needs to communicate with the Meta and Storage services, you must configure `spec.sslCerts` to set certificates for it when enabling mTLS between services.
```

Comment:
Please state clearly which yaml `spec.sslCerts` is in.

Reply:
done